src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Debug/LaunchSettingsProvider.cs

```diff
@@ -252,7 +264,9 @@ protected async Task UpdateProfilesAsync(string? updatedActiveProfileName)
                 // won't be called on F5 and the user will see a poor error message
                 if (launchSettingData.Profiles.Count == 0)
                 {
-                    launchSettingData.Profiles.Add(new LaunchProfileData() { Name = Path.GetFileNameWithoutExtension(_commonProjectServices.Project.FullPath), CommandName = RunProjectCommandName });
+                    var defaultLaunchProfile = _defaultLaunchProfile.Value;
+                    if (defaultLaunchProfile != null)
```

Comment:
Nit: The code is not consistent about this, but we generally prefer the `defaultLaunchProfile is not null` form over `defaultLaunchProfile != null` in new code.

Reply:
Fixed